webhook_to_fedora_messaging/endpoints/message.py

```diff
@@ -0,0 +1,36 @@
+from flask import Blueprint, Flask, request, Response, Request
+from ..database import db
+from ..models.service import Service
+from ..models.user import User
+from sqlalchemy_helpers import get_or_create
+from .util import not_found, success, bad_request, created, conflict, validate_request, create_event, query_fasjson
```

Comment:
@abompard It seems that @brngylni has a separate PR for the `util` module that is intended to fill the shoes here. As such the said `util` module does not exist in the targetted branch and hence, the inclusion of it would assume that a new file is created. @brngylni intends to merge #32 first i.e. the branch that has the `util` module shared by all of these endpoint modules. As such, I suggest @brngylni either include the `create_event` and `query_fasjson` in that PR or wait for that PR to be merged first before you rebase atop the newly-changed targetted branch with those additions to the `util` module in this PR.